packages/kit/types/index.d.ts

```diff
@@ -83,6 +83,8 @@ export interface Builder {
 	config: ValidatedConfig;
 	/** Information about prerendered pages and assets, if any. */
 	prerendered: Prerendered;
+	/** `true` if one or more routes have `export const config = ..` in it */
+	hasRouteLevelConfig: boolean;
```

Comment:
We can theoretically find out if there is through `createEntries`, though that might be wasteful sometimes, and it's just a little nicer to write an adapter using this boolean, which is why I added the API

Reply:
True! Having one config only would just become a "special" case